drivers/chibios/ws2812_spi.c

```diff
@@ -20,7 +20,7 @@
 #define NB_COLORS 3
 #define BYTES_FOR_LED (BYTES_FOR_LED_BYTE * NB_COLORS)
 #define DATA_SIZE (BYTES_FOR_LED * RGBLED_NUM)
-#define RESET_SIZE 200
+#define RESET_SIZE (1000 * WS2812_TRST_US / (2 * 1250))
```

Comment:
For reference I added the driver, i dont need talking down to on that it uses SPI to drive the leds.

Reply:
I have applied this suggestion, but as follows:
```c
#define RESET_SIZE (1000 * BYTES_FOR_LED_BYTE * WS2812_TRST_US / 1250)
```
The 1000 factor in the numerator and denominator are to ensure that we have a simple integer result; otherwise there are compile errors.

Reply:
Sorry @spidey3, going to have to back out e61b56a. I think you got there between where i updated my comment.

Reply:
@zvecr - sorry - I am confused.  Are you saying that the maths were correct in my original commit, before [e61b56a](https://github.com/qmk/qmk_firmware/commit/e61b56a2cfc7dfec9992a7a3af92afa50e5b8ec0)?

Reply:
yes